internal/buildscripts/packaging/choco/splunk-otel-collector/tools/chocolateyinstall.ps1

```diff
@@ -149,24 +149,43 @@ $packageArgs = @{
     validExitCodes = @(0)
 }
 
-Install-ChocolateyInstallPackage @packageArgs
+$use_msi_properties = [Version]$env:chocolateyPackageVersion -ge [Version]"0.99.0"

```

Comment:
Is this condition needed so users can use old collector versions with the new chocolatey release? Do we need to support that?

Reply:
Why are we comparing for >= "0.99"? 0.99 version of the chocolatey package was already released. Do we want to only use the msi properties code from new version of package going forward? Is this script not part of the packaging, as in, won't this change only go out in the next release of the package?

Reply:
Unfortunately yes: all the "scripts for install" are made this way and their version is not tied to the collector so for now it keeps the old code around until we deprecate those versions... some time in the future there will be a lot of code deletion :)

Reply:
> Is this script not part of the packaging, as in, won't this change only go out in the next release of the package?

Yes, this is correct only in the next published version. 

> Why are we comparing for >= "0.99"? 0.99 version of the chocolatey package was already released.

The chocolatey package has the same version as the MSI, they are packaged together - `0.99.0` is the first version of the MSI that supports the features needed for the install.

Reply:
> It may be best to keep support in place until we get sign off on the explicit support plan w.r.t versions

The choco package is tied to the collector version, so in principle we can remove the old code right now. 🤔 I'm not sure yet which way to go...

Reply:
Going with remove: it is just a small piece of code https://github.com/signalfx/splunk-otel-collector/pull/4755/commits/5d54c7e99431c97bef37cd298288b5a5b32b5050